docs/fflags.adoc

```diff
@@ -0,0 +1,16 @@
+== Feature Flags
+
+The following system properties are available for toggling or specifying particular features to turn on or off.
+
+* `com.cloudbees.plugins.credentials.UseOwnPermission` - Enables the Credentials/UseOwn permission type which allows a
```

Comment:
BTW https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
You may have noticed me doing that elsewhere in the adoc files ;)

Reply:
Fixed.